### DISCUSSION TITLE
Connected to [missing {{title}} value]

### DISCUSSION THREAD

**Discussion by mdesousa** [WalletConnect/walletconnect-monorepo#3311] [Developer Support] [answered]
Hi, i'm developing a web3 application using react + web3modal. When I connect the MetaMask Mobile wallet to the app, I see a "Connected to [missing {{title}} value]" message. The wallet works correctly, but i'm wondering how to provide the title value?

I have tried the following:
* completed the configuration in the WalletConnect dashboard for my project
* configured a manifest.json for my app

However, the application title is still not shown in the wallet. Thanks.

**Comment by glitch-txs** [comment]
Hi, do you have a title in the head tag?

**Comment by mdesousa** [comment]
hi @glitch-txs , thanks for your answer. yes, we have a title set in the head tag.

**Reply by glitch-txs** [reply]
- is your repo public? or have any minimal reproducible example we can test? Thanks
- Also have you check on our examples? https://github.com/WalletConnect/web3modal-examples
- Which wallet are you testing with?

**Reply by mdesousa** [reply]
i am using the mobile metamask wallet (v 7.3.0).
no, my repo isn't public. i was trying to get the `web3modal-examples` to work since they would be great to reproduce the issue. i have followed the readme instructions (including setting .env.local with our walletconnect project id). however, they are giving me errors (i'm using master branch):
* tried `webmodal-wagmi-react` but i get a "Error: No wagmi config found" as soon as the page opens
* tried `webmodal-wagmi-html`. the page opens and i see the "Connect" button. but when i click it it just hangs while "Connecting..."... there are errors in the console: "global is not defined"

do these examples work for you @glitch-txs ? should i try from a different branch?

**Reply by glitch-txs** [reply]
First issue is fixed with this [PR](https://github.com/WalletConnect/web3modal-examples/pull/39)
Second one needs polyfill

I was able to reproduce the issue, and seems to be comming from MetaMask side, kindly could you open an issue on MetaMask mobile repo? We'll try to look farther into this. Thanks for reporting 🤝

**Reply by glitch-txs** [reply]
just saw your [open issue](https://github.com/MetaMask/metamask-mobile/issues/6933), thanks for reporting

**Reply by mdesousa** [reply]
ah great @glitch-txs , thanks…. I was about to share that link. please feel free to share any additional details in that issue for the metamask team 👍

**Reply by mdesousa** [reply]
@glitch-txs another question: do you know if the source code for the web3modal.com website is in a public repo by any chance? I’m asking because the metamask mobile wallet displays the title correctly for the web3modal in that page. I wanted to take a look at the configuration… thanks

**Reply by glitch-txs** [reply]
Good catch, will take a look

**Reply by glitch-txs** [reply, marked as answer]
it's catching the description in the website metatag, also tested on our examples repo

Something like this in the head should work:
`<meta name="description" content="any descrition here" />`

**Reply by mdesousa** [reply]
thanks @glitch-txs that solved it 👍 
do you know by any chance where the icon comes from? (the one that appears in the "Connect to this site?" prompt)
we have an icon configured in the head, but it's not picking that up: `<link rel="icon" type="image/png" href="/myicon.png" />`

**Reply by glitch-txs** [reply]
Mmm that should work. Are you able to get the image if you go to your domain + /myicon.png? Is this only on MetaMask? Also what's the image size?

here's the source code btw: https://github.com/WalletConnect/walletconnect-utils/blob/dbad8f5653b6ce70e6b32fe8dd9326520538456b/browser/window-metadata/src/index.ts#L21

**Reply by mdesousa** [reply]
@glitch-txs yes the image is reachable. size is 32x32… I’ll read through that source, thanks
